services/security/security.go

```diff
@@ -243,7 +243,7 @@ type Certificate struct {
 	// X509Thumbprint - READ-ONLY; Thumbprint of the certificate. (a URL-encoded base64 string)
 	X509Thumbprint *string `json:"x5t,omitempty"`
 	// Cer - CER contents of x509 certificate.
```

Comment:
can you also update the comment saying this is base64 encoded string

Reply:
Done.